src/site/views/event/tmpl/default.php

```diff
@@ -101,7 +102,73 @@
 							</tr>
 							</tfoot>
 						</table>
+						<table class="table table-hover">
+							<thead>
+							<tr>
+								<th>University</th>
+								<th>Tickets Sold</th>
+							</tr>
+							</thead>
+							<tbody>
+							<?php
+							$universityCounts = array();
+
+							foreach ($eventAttendees as $person)
+							{
+								$universityCounts[$person['Uni']] += 1;
+							}
+							arsort($universityCounts);
+							foreach ($universityCounts as $uni => $count)
+							{
+								echo "<tr>\n";
+								echo "<td>{$uni}</td>";
+								echo "<td>{$count}</td>";
+								echo "</tr>\n";
+							}
+							?>
+							</tbody>
+						</table>
+
+						<?php
+						  // Only want to show the full attendees table when the club committees actually need it
+						  $registrationClosed = time() > (24 * 60 * 60 + strtotime($item->event_date_close));
```

Comment:
Looks like 1 day after the ticket sales close. I would say this should be available from when ticket sales open to the end of the event. For some events, tickets are still being sold on the Friday of and President pack often released before ticket sales close. Would be too late to buy food and plan which houses everyone will be in etc.

Reply:
Yeah I was unsure on how useful these timings would be. I'll change to viewing from ticket open.